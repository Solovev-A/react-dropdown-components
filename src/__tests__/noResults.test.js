import React from 'react';
import { render, screen } from "@testing-library/react";
import 'jest-styled-components';

import NoResults from './../lib/components/noResults';

describe('NoResults component', () => {
    it('renders correctly', () => {
        render(<NoResults />);
        expect(screen.getByText(/совпадений не найдено/i)).toMatchSnapshot();
    });
});