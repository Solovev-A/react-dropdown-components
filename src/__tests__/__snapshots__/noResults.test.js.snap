// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults component renders correctly 1`] = `
.c0 {
  padding: .375rem .75rem;
  display: block;
  white-space: nowrap;
  cursor: pointer;
}

.c1 {
  cursor: default;
}

<span
  class="c0 c1"
>
  Совпадений не найдено
</span>
`;
